LondonBritishLibrary/orient/BLorient2082.xml

```diff
@@ -0,0 +1,375 @@
+                                </msItem>
+                                <msItem xml:id="p1_i1.3">
+                                    <locus from="25ra" to="45vb"/>
+                                    <title ref="LIT2375Synaxa#Miyazya"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.4">
+                                    <locus from="46ra" to="71va"/>
+                                    <title ref="LIT2375Synaxa#Genbot"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.5">
+                                    <locus from="72ra" to="96vb"/>
+                                    <title ref="LIT2375Synaxa#Sane"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.6">
+                                    <locus from="97ra" to="130rb"/>
+                                    <title ref="LIT2375Synaxa#Hamle"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.7">
+                                    <locus from="131ra" to="154rb"/>
+                                    <title ref="LIT2375Synaxa#Nahase"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.8">
+                                    <locus from="155ra" to="159va"/>
+                                    <title ref="LIT2375Synaxa#Pagwemen"/>
+                                </msItem>
+                                <colophon xml:id="coloph1">
+                                    <locus target="#159vb"/>
+                                    <note>The note states that this manuscript was bought by 
+                                        <persName role="donor" ref="PRS14454EhataKrestos">ʾƎḫǝta Krǝstos</persName> from
+                                        <persName role="owner" ref="PRS14455AsqaGiyorgis">ʾAṣqa Giyorgis</persName> and offered to the 
+                                        <placeName ref="LOC000000000000000000000000000">church of St George</placeName> in the tenth year of the 
+                                        reign of <persName ref="PRS4015Fasilada">Fāsiladas</persName>, i.e. <date when="1642">1642 AD</date>.</note>
+                                    <q xml:lang="gez">ዛቲ፡ ስንክሳር፡ ዘ<persName role="donor" ref="PRS14454EhataKrestos">እኅተ፡ ክርስቶስ፡</persName> 
+                                        ዘተሣየጠት፡ በ፯ወቄት፡ እም<persName role="owner" ref="PRS14455AsqaGiyorgis">አጽቀ፡ ጊዮርጊስ፤</persName> 
+                                        ወመድኅንሂ፡ <persName ref="PRS14456WalattaDengel"><roleName type="title">ወይዘሮ፡</roleName>ወለተ፡ ድንግል፤</persName> 
+                                        ወምስክርሂ፡ <persName ref="PRS14457WaldaHawWS">ወልደ፡ ሐዋርያት፡ ወልደ፡ ሰንቡል፤</persName>
+                                        ወ<persName ref="PRS14458Zamaryam">ዘማርያም፡</persName> ዘነበረ፡ ወዓሊሁ፤ 
```

Comment:
You do not need to encode everything.

Reply:
I created PRS records for each name that I safely identified as personal names. I will delete the markup for the other for which I am not certain.